packages/reports/addon/components/navi-date-picker.js

```diff
@@ -173,29 +156,33 @@ export default Component.extend({
     }
 
     return moment(lastTime).isSame(newDate);
-  },
-
-  actions: {
-    /**
-     * Action sent whenever user makes a selection
-     * Converts the selected Date into a moment and
-     * passes the action on
-     *
-     * @action
-     * @param {Date} newDate - selected calendar date
-     */
-    changeDate(newDate) {
-      this._highlightSelection();
-      const handleUpdate = get(this, 'onUpdate');
-
-      // Don't do anything if the date is the same as the last time action was called
-      if (this._isNewDateValue(newDate)) {
-        return;
-      }
+  }
 
-      // Convert date to start of time period
-      let dateTimePeriod = getIsoDateTimePeriod(get(this, 'dateTimePeriod'));
-      if (handleUpdate) handleUpdate(moment(newDate).startOf(dateTimePeriod));
+  /**
+   * Action sent whenever user makes a selection
+   * Converts the selected Date into a moment and
+   * passes the action on
+   *
+   * @action
+   * @param {Date} newDate - selected calendar date
+   */
+  @action
+  changeDate(newDate) {
+    // Don't do anything if the date is the same as the last time action was called
+    if (this._isNewDateValue(newDate)) {
```

Comment:
Seems like the name of `_isNewDateValue` is backwards since it's returning true if the passed date is NOT a new date value.